osiris/query/server.go

```diff
@@ -103,33 +104,24 @@ func serve(ctx *cli.Context) error {
 }
 
 func main() {
+	cliFlags := []cli.Flag{}
+	cliFlags = append(cliFlags, libcli.CommonCliOptions(libcli.ServicePort, libcli.ServiceLogLevel)...)
```

Comment:
If i understand it correctly it might be more readable if also ServiceLogLevel had its own line. It is the new potion to enable debug right?

Reply:
Looks fine to me